src/calendarevent.cpp

```diff
@@ -130,7 +130,11 @@ CalendarEvent::Recur CalendarEvent::recur() const
 
 QDateTime CalendarEvent::recurEndDate() const
 {
+#if (QT_VERSION >= QT_VERSION_CHECK(5, 14, 0))
+    return QDateTime(mData->recurEndDate.endOfDay());
```

Comment:
Here also, you can remove the `QDateTime()` since `.endOfDay()` is returning a QDateTime object.